packages/experimental/src/experimental-list/task-item.tsx

```diff
@@ -207,6 +211,20 @@ export const TaskItem: React.FC< TaskItemProps > = ( {
 									) }
 								</Button>
 							) }
+							{ onDelete && completed && (
```

Comment:
Depending on any new options coming down the pipe, it might be worth changing this into an array of options with a title and callback, so that we wouldn't have to update the `task-item` for any future ellipsis options.

Reply:
Thanks @louwie17, that's a good idea! I'll merge this PR and create a follow-up issue with this suggestion.